drift-transport-netty/src/main/java/com/facebook/drift/transport/netty/client/DriftNettyClientConfig.java

```diff
@@ -59,6 +60,11 @@
     private long sessionCacheSize = 10_000;
     private Duration sessionTimeout = new Duration(1, DAYS);
 
+    private Boolean connectionPoolEnabled;
+    private Integer connectionPoolMaxSize;
+    private Integer connectionPoolMaxConnectionsPerDestination;
```

Comment:
Do we need wrapper objects here. Would primitives suffice

Reply:
Wrappers are needed so the properties can be set to null. If the per-client property is set to null we fallback to the global setting.